core/src/main/scala/cats/data/Nested.scala

```diff
@@ -133,13 +133,20 @@ private[data] sealed abstract class NestedInstances9 extends NestedInstances10 {
     }
 }
 
-private[data] sealed abstract class NestedInstances10 {
+private[data] sealed abstract class NestedInstances10 extends NestedInstances11 {
   implicit def catsDataInvariantForNestedContravariant[F[_]: Invariant, G[_]: Contravariant]: Invariant[Nested[F, G, ?]] =
     new NestedInvariant[F, G] {
       val FG: Invariant[λ[α => F[G[α]]]] = Invariant[F].composeContravariant[G]
     }
 }
 
+private[data] sealed abstract class NestedInstances11 {
```

Comment:
I think you can put the `catsDataTraverse1ForNested` inside `NestedInstances`.

As you can never have a `TraverseFilter` and a `Traverse1` for a data type at the same time, I don't think that the composition `Traverse` and `TraverseFilter` versus the composition of `Traverse1` and `Traverse1` can be ambiguous.